{"name": "test2", "status": "failed", "statusDetails": {"message": "AssertionError: assert None == 'qeehfjejwjwjej11sss@22'", "trace": "self = <cases.test02.Test object at 0x00000199B9FE8320>, get_token = None\n\n    def test2(self, get_token):\n        token = 'qeehfjejwjwjej11sss@22'\n        print(\"【执行test02.py-Test类-test2用例,获取get_token：%s】\" % get_token)\n>       assert get_token == token\nE       AssertionError: assert None == 'qeehfjejwjwjej11sss@22'\n\ncases\\test02.py:18: AssertionError"}, "start": 1616990359618, "stop": 1616990359618, "uuid": "2e6cff89-6afb-4d82-b5f9-2e1b48a7d1ef", "historyId": "88ab98f351d58eb733e57db34394d628", "testCaseId": "c06aa9ef4d49381097c14c1bfc0c4c50", "fullName": "cases.test02.Test#test2", "labels": [{"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test02"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "DESKTOP-MI5M7JM"}, {"name": "thread", "value": "11444-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test02"}]}